src/yokozuna.erl

```diff
@@ -23,6 +23,51 @@
 
 -type fold_fun() :: fun(([term()], Acc::term()) -> Acc::term()).
 
+-behaviour(riak_kv_update_hook).
+
+%% riak_kv_update_hook behaviors
+-export([
+    update/3,
+    update_binary/5,
+    requires_existing_object/1,
+    should_handoff/1
+]).
+
+%%%===================================================================
+%% riak_kv_update_hook behavior
+%%%===================================================================
+
+%% @doc see riak_kv_update_hook:update/3
```

Comment:
Of course, we don't actually generate any HTML docs from these comments (do we?) so this is totally discretionary.